src/ui/screens/settings/ViewReset.qml

```diff
@@ -169,7 +172,13 @@ ViewFullScreen {
                     text: MZI18n.ResetSettingsConfirmResetModalResetButtonLabel
                     colorScheme: MZTheme.theme.redButton
 
-                    onClicked: VPN.hardResetAndQuit()
+                    onClicked: {
+                        Glean.interaction.resetVpnSelected.record({
+                            screen: root.telemetryScreenId,
+                        });
+
+                        VPN.hardResetAndQuit()
+                    }
```

Comment:
@brizental is this metric possible to test with the VPN quitting right after the event is recorded?

Reply:
Should be possible... because the metric is stored and will be there on the next start. So just quitting and reopening should do it. However, because the main ping is submitted on shutdown maybe it's not... 

This would be possible if we had `test_before_next_submit`. https://mozilla-hub.atlassian.net/browse/VPN-3338 Let's block testing on this ticket and increase the priority of it to medium -- unless you want to go ahead and implement it :eyes: